benchmarks/microbenchmarks/operatorbench.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import click
+import numpy as np
+import torch
+import triton
+from operator_inp_utils import OperatorInputsLoader
+
+import torchinductor
+from torchdynamo.optimizations.backends import cudagraphs_inner
+from torchdynamo.testing import same
+from torchinductor.compile_fx import compile_fx
+from torchinductor.utils import gen_gm_and_inputs
+
+aten = torch.ops.aten
+
+
+def compute_speedups(repeats, models, example_inputs):
+    expected = models[0](*example_inputs)
+    for model in models[1:]:
+        actual = model(*example_inputs)
+        assert same(actual, expected), expected[0] - actual[0]
+
+    timings = np.zeros((repeats, len(models)), np.float64)
+    for rep in range(repeats):
+        # interleave the runs to handle frequency scaling and load changes
+        for m, model in enumerate(models):
+            # do_bench() clears L2 cache to hide the latency of CPU launch time
+            # along with cuda synchronization
+            median_ms, _, _ = triton.testing.do_bench(lambda: model(*example_inputs))
+            timings[rep, m] = median_ms
+    return np.median(timings, axis=0)
+
+
+def microbenchmark(target, args, kwargs, dtype):
+    gm, gm_args = gen_gm_and_inputs(target, args, kwargs)
+    compiled_fn = compile_fx(gm, gm_args)
+    cudagraphs_eager = cudagraphs_inner(gm, gm_args, copy_outputs=False)
+    cudagraphs_jit = cudagraphs_inner(
+        torch.jit.trace(gm, gm_args), gm_args, copy_outputs=False
+    )
+
+    repeats = 3
```

Comment:
Can we just have a fast correctness checking mode? It will improve our improve test coverage, and may also help us to identify if any of those model accuracy error is real.

Reply:
Will do as follow up, when I looked into this previously I think there are are a lot of nan-handling errors, which I think the details of are a little bit tricky what pytorch guarantees (will do non-empty input generation to avoid nan errors).